lib/msal-common/src/error/ClientConfigurationError.ts

```diff
@@ -81,6 +81,10 @@ export const ClientConfigurationErrorMessage = {
         code: "untrusted_authority",
         desc: "The provided authority is not a trusted authority. Please include this authority in the knownAuthorities config parameter."
     },
+    invalidAzureCloudInstance: {
+        code: "invalid_azure_cloud_instance",
+        desc: "Invalid AzureCloudInstance provided. Please refer MSAL JS docs for valid values"
```

Comment:
This should probably link to relevant docs.

Reply:
Done